primitives/src/locktime/relative.rs

```diff
@@ -281,30 +280,49 @@ impl LockTime {
     /// # use bitcoin_primitives::Sequence;
     /// # use bitcoin_primitives::locktime::relative::{LockTime, Height, Time};
     ///
-    /// let intervals: u16 = 70; // approx 10 hours;
-    /// let lock = Sequence::from_512_second_intervals(intervals).to_relative_lock_time().expect("valid time");
-    /// assert!(lock.is_satisfied_by_time(Time::from_512_second_intervals(intervals + 10)).expect("a time"));
+    /// // ["The timespan starts from the median-time-past of the output’s previous block, and ends at the MTP of the previous block."](https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki)
+    ///
+    /// // This is the 16 bit integer value that is multiplied by 512, equivalently shifting up by 9 bits ,to get the time in seconds as defined by Bip68.
+    /// let timespan: u16 = 70;
+    ///
+    /// // Random, but valid, unix timestamp. the value scheme used in blocks timestamp.
+    /// let prevout_mtp = 1_000_000_000 as u32;
+    ///
+    /// // A unix timestamp 512 seconds after the timespan since the MTP of the prevout block. (1_000_000_000 + (512 * 70)).
+    /// let actual_time = 1_000_000_000 + (512 * 71) as u32;
+    ///
+    /// // Cast the LockTime from a Sequence containing the timespan.
+    /// let lock = Sequence::from_512_second_intervals(timespan).to_relative_lock_time().expect("valid timespan");
+    /// assert!(lock.is_satisfied_by_time(actual_time, prevout_mtp).expect("The Actual Time and the prevout MTP."));
     /// ```
     #[inline]
     #[cfg_attr(all(test, mutate), mutate)]
-    pub fn is_satisfied_by_time(&self, time: Time) -> Result<bool, IncompatibleTimeError> {
+    pub fn is_satisfied_by_time(
+        &self,
+        actual_time: u32,
+        prevout_mtp: u32,
+    ) -> Result<bool, IncompatibleTimeError> {
```

Comment:
Notice that the inputs changed to be `u32` integers since the implementation validates agains u32 values and not Locktimes.

Reply:
The to validate against doesnt fit on u16. and they dont need to since they are external to the sequence.